config/config_test.go

```diff
@@ -129,3 +129,23 @@ func TestParse(t *testing.T) {
 		}
 	}
 }
+
+func TestParseFields(t *testing.T) {
+	config, err := NewConfig(options.Options{Config: "../test_data/test_configs/fields_test.toml"})
+	if err != nil {
+		t.Error(err)
+	}
+
+	expectedNumOfRules := 2
+	if len(config.Rules) != expectedNumOfRules {
+		t.Fatalf("expected %v rules", expectedNumOfRules)
+	}
+
+	if config.Rules[0].ReportGroup != 0 {
```

Comment:
**not blocking:** super minor but IMO it would be a slightly more robust to use a description reference  rather than index; i.e.

```go
idx := sort.Search(len(config.Rules), func(i int) bool {
        return config.Rules[i].Description ==  "Some Groups without a reportGroup"
    })

if config.Rules[idx].ReportGroup != 0 {
```

From a more general-musing standpoint, that's one issue I have with dedicated fixtures outside the specific test cases. This wouldn't matter as long as no other code uses `fields_test.toml`, but if another relies on the fixture and tweaks something, this could inadverently break with the relative index reference.